go.mod

```diff
@@ -12,12 +12,12 @@ require (
 	github.com/containernetworking/cni v1.1.2
 	github.com/containernetworking/plugins v1.1.1
 	github.com/containers/buildah v1.26.1-0.20220716095526-d31d27c357ab
-	github.com/containers/common v0.48.1-0.20220720145307-26032247af78
+	github.com/containers/common v0.49.1-0.20220729170140-87fab4b7019a
```

Comment:
We should get a common release out and vendored before 8.7/9.1

Reply:
https://github.com/containers/common/pull/1107#pullrequestreview-1056811028

Reply:
bumped to `v0.49.1`